x-pack/plugins/secops/public/components/flyout/index.test.tsx

```diff
@@ -36,7 +36,7 @@ describe('Flyout', () => {
     test('it renders the default flyout state as a button', () => {
       const wrapper = mount(
         <ReduxStoreProvider store={store}>
-          <ThemeProvider theme={{ eui: euiVars }}>
+          <ThemeProvider theme={() => ({ eui: euiDarkVars, darkMode: true })}>
```

Comment:
Optional, you could save repetition by just moving this into a function at the top called theme:

```ts
 const theme = () => ({ eui: euiDarkVars, darkMode: true });
```

And then just use it throughout this file to cut down on the semantic noise.